.meteor/packages

```diff
@@ -69,6 +69,7 @@ kadira:flow-router-ssr
 matb33:collection-hooks
 meteorhacks:ssr
 meteorhacks:subs-manager
+natestrauser:select2
```

Comment:
We should be avoiding introducing new atmosphere packages.  Can't this be used as an npm import?